### ISSUE TITLE
Scaling of surfaces and their subsurfaces

### ISSUE THREAD

**Issue by hirseboy** [ghorwin/SIM-VICUS#371]
When surfaces are scaled, their subsurfaces also need to get treated somehow.

There are two possebilities:
1) when surfaces and subsurfaces are both selected, scale also subsurfaces according to scaling factors. 
This is not a problem and will be implemented by me.
2) But if sub surfaces are not selected -> what scaling is then expected? Should the subsurfaces always stay at their position as before?

**Comment by ghorwin** [comment]
yes - if not selected (or subsurfaces have been de-selected) they remain positioned as before, i.e. relative to the local coordinate system of the surface. If the origin is moved as part of the scaling operations, so are the subsurfaces (but actually their local coordinates remain unchanged).
